opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchBoolPrefixQueryTest.java

```diff
@@ -45,7 +45,10 @@ static Stream<List<Expression>> generateValidData() {
             dsl.namedArgument("fuzzy_transpositions", DSL.literal("true")),
             dsl.namedArgument("fuzzy_rewrite", DSL.literal("constant_score")),
             dsl.namedArgument("minimum_should_match", DSL.literal("3")),
-            dsl.namedArgument("boost", DSL.literal("1"))
+            dsl.namedArgument("boost", DSL.literal("1")),
+            dsl.namedArgument("analyzer", DSL.literal("simple")),
+            dsl.namedArgument("operator", DSL.literal("Or")),
+            dsl.namedArgument("operator", DSL.literal("and"))
```

Comment:
can it return both `and` and `or`?

Reply:
No -- it's either AND or ... OR.

 See https://github.com/MaxKsyunz/OpenSearch/blob/917f9c2699f18fb6d338b93d7962d664e5e2bc6e/server/src/main/java/org/opensearch/index/query/MatchBoolPrefixQueryBuilder.java#L152